dev/docker-compose.dev.full.test.yml

```diff
@@ -0,0 +1,121 @@
+  curatorui:
+    command: "npm run start-noenv"
+    volumes:
+      - "../verification/curator-service/ui/src:/usr/src/app/verification/curator-service/ui/src"
+    environment:
+      # We can't use curator:3001 here because that's an internal DNS,
+      # not accessible from the user's browser.
+      # In production simply /auth/google would work.
+      REACT_APP_LOGIN_URL: "http://localhost:3001/auth/google"
+      REACT_APP_PUBLIC_MAPBOX_TOKEN: "${REACT_APP_PUBLIC_MAPBOX_TOKEN}"
+      REACT_APP_POLICY_PUBLIC_ID: "${REACT_APP_POLICY_PUBLIC_ID}"
+      REACT_APP_COOKIE_CONSENT_PUBLIC_ID: "${REACT_APP_COOKIE_CONSENT_PUBLIC_ID}"
+      ENABLE_LOCAL_AUTH: "true"
+  localstack:
+    image: localstack/localstack-full
+    ports:
+      - "53:53"
+      - "443:443"
+      - "4510:4510"
+      - "4566:4566"
+      - "4571:4571"
+      - "8080:8080"
+    volumes:
+      - "/var/run/docker.sock:/var/run/docker.sock"
+    environment:
+      AWS_ACCESS_KEY_ID: "fake"
+      AWS_SECRET_ACCESS_KEY: "fake"
+      LOCALSTACK_API_KEY: "${LOCALSTACK_API_KEY}"
+      SERVICES: s3,ec2,ses,batch,events,iam,lambda,ecr
+      DEBUG: 1
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:4566/health"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+      start_period: 10s
+  mock-source-data:
+    build:
+      context: ./
+      dockerfile: Dockerfile-mock-source-data
+    ports:
+      - "5000:5000"
+    volumes:
+      - "../ingestion/functions:/ingestion/functions"
+    environment:
+      MOCK_SOURCE_DATA_HOST: "0.0.0.0"
+      MOCK_SOURCE_DATA_PORT: 5000
+      MOCK_SOURCE_DATA_SERVICE: "http://mock-source-data"
+      AWS_ACCESS_KEY_ID: "fake"
+      AWS_SECRET_ACCESS_KEY: "fake"
+      AWS_DEFAULT_REGION: "us-east-1"
+      AWS_ENDPOINT: "http://localstack:4566"
+      RETRIEVAL_BUCKET_NAME: "epid-sources-raw"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:5000/health"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+      start_period: 10s
+  test:
+    build:
+      context: ./
+      dockerfile: Dockerfile-test
+    depends_on:
+      - mongo
+      - mock-source-data
+      - localstack
+    ports:
+      - "4444:4444"
+    volumes:
+      - "../ingestion/functions:/ingestion/functions"
+      - "/var/run/docker.sock:/var/run/docker.sock"
+    environment:
+      AWS_ACCESS_KEY_ID: "fake"
+      AWS_SECRET_ACCESS_KEY: "fake"
+      AWS_DEFAULT_REGION: "us-east-1"
+      AWS_ENDPOINT: "http://localstack:4566"
+      DATA_BUCKET_NAME: "covid-19-data-export"
+      CACHE_BUCKET_NAME: "covid-19-cache"
+      RETRIEVAL_BUCKET_NAME: "epid-sources-raw"
+      BATCH_QUEUE_NAME: "ingestion-queue"
+      SES_EMAIL_ADDRESS: "info@global.health"
+      ECR_REPOSITORY_NAME: "gdh-ingestor"
+      MOCK_SOURCE_DATA_SERVICE: "http://mock-source-data"
+      MOCK_SOURCE_DATA_PORT: 5000
+      CLIENT_EMAIL: "test_account@global.health"
+      CLIENT_PASSWORD: "JBCaL8QfX8eTzS9"
```

Comment:
If it's a local only / testing account, then we can not use passwords as well.

Reply:
It appeared necessary to create a new user, and I needed to disable email verification locally.
At any rate, this also tests the sign up and sign in pages, which is desirable.